guidelines/terms/20/contrast-ratio.html

```diff
@@ -15,20 +15,20 @@
    
    <p class="note">Contrast ratios can range from 1 to 21 (commonly written 1:1 to 21:1).</p>
    
-   <p class="note">Because authors do not have control over user settings as to how text is rendered
-      (for example font smoothing or anti-aliasing), the contrast ratio for text can be
+   <p class="note">Because authors do not have control over user settings as to how text and non-text content is rendered
+      (for example font smoothing or anti-aliasing), the contrast ratio can be
       evaluated with anti-aliasing turned off.
    </p>
    
    <p class="note">For the purpose of Success Criteria 1.4.3 and 1.4.6, contrast is measured with respect
-      to the specified background over which the text is rendered in normal usage. If no
-      background color is specified, then white is assumed.
+      to the specified background over which the content is rendered in normal usage. If no
+      background color is specified, then white is assumed. If no
+      foreground color is specified, then black is assumed.
```

Comment:
as 1.4.3 and 1.4.6 *are* specifically about text, there's no sense here changing things to be more generic